allennlp/commands/train.py

```diff
@@ -300,8 +306,15 @@ def train_model(params: Params,
     # Now tar up results
     archive_model(serialization_dir, files_to_archive=params.files_to_archive)
 
+    logger.info("Loading the best epoch weights.")
+    best_model_state_path = os.path.join(serialization_dir, 'best.th')
```

Comment:
instead of hardcoding `best.th`, maybe import `_DEFAULT_WEIGHTS` from model? https://github.com/allenai/allennlp/blob/master/allennlp/models/model.py#L25

Reply:
Good point. I actually looked for a constant with the strict semantic of "best weights file name" before hard-coding it. I saw ``_DEFAULT_WEIGHTS`` but: 1) I think it has another semantic and 2) it's "declared" as "private". Furthermore, ``"best.th"`` is hard-coded elsewhere, so I hard-coded it. I think we could introduce a constant ``BEST_WEIGHTS`` or better (though longer) ``BEST_WEIGHTS_FILENAME``.

Reply:
Actually, looking again at model.py, I think that ``_DEFAULT_WEIGHTS`` could be safely renamed as ``BEST_WEIGHTS`` or ``BEST_WEIGHTS_FILENAME``.